### PR TITLE
[zh-cn]: translate for AbortController

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

File: files/zh-cn/web/api/abortcontroller/index.md [hunk 1]
```diff
@@ -78,4 +80,4 @@ function fetchVideo() {
 ## 参见
 
 - [Fetch API](/zh-CN/docs/Web/API/Fetch_API)
-- [Abortable Fetch](https://developer.chrome.com/blog/abortable-fetch/) by Jake Archibald
+- [可中止的 Fetch](https://developer.chrome.com/blog/abortable-fetch/) by Jake Archibald
```
Comment (T34-active) [inline files/zh-cn/web/api/abortcontroller/index.md:83]:
```suggestion
- Jake Archibald 的[可中止的 Fetch](https://developer.chrome.com/blog/abortable-fetch/)
```

File: files/zh-cn/web/api/abortcontroller/index.md [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+- {{domxref("AbortController.AbortController()", "AbortController()")}}
   - : 创建一个新的 `AbortController` 对象实例。
 
-## 属性
+## 实例属性
 
-- {{domxref("AbortController.signal")}} {{readonlyInline}}
-  - : 返回一个 {{domxref("AbortSignal")}} 对象实例，可以用它与一个 DOM 请求进行通信或者中止该请求。
+- {{domxref("AbortController.signal")}} {{ReadOnlyInline}}
+  - : 返回一个 {{domxref("AbortSignal")}} 对象实例，可以用它与一个 异步操作进行通信或者中止该请求。
```
Comment (yin1999) [inline files/zh-cn/web/api/abortcontroller/index.md:22]:
```suggestion
  - : 返回一个 {{domxref("AbortSignal")}} 对象实例，可以用它来和异步操作进行通信或者中止这个操作。
```

File: files/zh-cn/web/api/abortcontroller/index.md [hunk 3]
```diff
@@ -1,28 +1,30 @@
...
 
-## 方法
+## 实例方法
 
 - {{domxref("AbortController.abort()")}}
-  - : 中止一个尚未完成的 Web（网络）请求。这能够中止 [fetch](/zh-CN/docs/Web/API/fetch) 请求及任何响应体的消费和流。
+  - : 中止一个尚未完成的异步操作。这能够中止 [fetch](/zh-CN/docs/Web/API/fetch) 请求及任何响应体的消费和流。
```
Comment (yin1999) [inline files/zh-cn/web/api/abortcontroller/index.md:27]: response bodies 和 streams 应并列

```suggestion
  - : 中止一个尚未完成的异步操作。这能够中止 [fetch 请求](/zh-CN/docs/Web/API/fetch)及任何响应体和流的使用。
```

File: files/zh-cn/web/api/abortcontroller/index.md [hunk 4]
```diff
@@ -1,28 +1,30 @@
 ---
 title: AbortController
 slug: Web/API/AbortController
+l10n:
+  sourceCommits: 15f0b5552bc9c2ea1f32b0cd5ee840a7d43c887e
```
Comment (yin1999) [inline files/zh-cn/web/api/abortcontroller/index.md:5]:
```suggestion
  sourceCommit: 15f0b5552bc9c2ea1f32b0cd5ee840a7d43c887e
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**